api/v1alpha1/environment_types.go

```diff
@@ -660,7 +660,3 @@ type EnvironmentList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Environment `json:"items"`
 }
-
-func init() {
-	SchemeBuilder.Register(&Environment{}, &EnvironmentList{})
-}
```

Comment:
I'm not sure I understand why this goes away.

Reply:
Similar question for `Promotion` and `PromotionPolicy`

Reply:
`go-to-protobuf` fails with this `init` function (error shows that it can't recognizes `SchemeBuilder`), so delegated registration process to `addKnownTypes` function 👀

Reply:
Gotcha! 👍